### DISCUSSION TITLE
.gif creating gradient lines not present in original

### DISCUSSION THREAD

**Discussion by coltssite** [ImageMagick/ImageMagick#2078] [Help] [answered]
I am trying to convert a .pdf to a .gif. However, when I convert using:

`/usr/local/bin/convert -density 300 -loop 0 -background white -alpha remove *.pdf *.gif`

ImageMagick creates gradient lines that are not present in the original pdf. I've played around with options, but can't seem to narrow down the problem. I would greatly appreciate any input!

Version: ImageMagick 7.0.8-53 Q16 x86_64 2019-07-09 
OS: Mac Catalina

![*](https://user-images.githubusercontent.com/65022490/83328278-c1b4b280-a236-11ea-91b2-d0051a9561ed.gif)
[electron_cloud.pdf](https://github.com/ImageMagick/ImageMagick/files/4705612/electron_cloud.pdf)

**Comment by dlemstra** [comment]
Can you reproduce the issue with the latest version of ImageMagick?

**Comment by fmw42** [comment, marked as answer]
The lines are in your alpha channel.

`convert -density 300 electron_cloud.pdf[5] -alpha extract x.png`

![x](https://user-images.githubusercontent.com/13458218/83333967-38ae7300-a258-11ea-963e-026fe7388ffd.png)



`convert -density 300 electron_cloud.pdf[5] -alpha off y.png`

![y](https://user-images.githubusercontent.com/13458218/83333970-3c41fa00-a258-11ea-8153-95826ed9314d.png)

**Reply by coltssite** [reply]
Thank you! Is there any way to persevere the white background as well?

**Reply by fmw42** [reply]
One can apply a median filter to the alpha channel and remove the lines as follows:

`convert -density 300 -delay 10 electron_cloud.pdf -channel a -statistic median 3x3 +channel -background white -alpha background -alpha remove -loop 0 y.gif`

![y](https://user-images.githubusercontent.com/13458218/83359700-cbbadc00-a330-11ea-8ff8-aab3e33f2f8f.gif)

**Reply by coltssite** [reply]
Perfect thank yoU!
